ddsketch/mapping.py

```diff
@@ -0,0 +1,177 @@
+# Unless explicitly stated otherwise all files in this repository are licensed
+# under the Apache License 2.0.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2020 Datadog, Inc.
+
+"""A mapping between values and integer indices that imposes relative accuracy
+guarantees. Specifically, for any value minIndexableValue() < value < maxIndexableValue
+()}, implementations of {@link IndexMapping} must be such that {@code value(index(v))}
```

Comment:
I think `@link` and `@code` don't make sense in Python docstrings.

Reply:
🤦 I must have gotten distracted after reformatting the first line.